RELEASES.md

```diff
@@ -0,0 +1,188 @@
+# Swift for TensorFlow Release Notes
+
+## Version 0.3
+
+### Overview
+
+This is the second public release of Swift for TensorFlow, available across
+Google Colaboratory, Linux and macOS. The focus is improving overall stability
+and refining APIs.
+
+### Notebook Environments (Colab and Jupyter)
+
+* Install SwiftPM packages using `%install` directives. See [documentation in
+  README](https://github.com/google/swift-jupyter#install-directives).
+  ([swift-jupyter#45](https://github.com/google/swift-jupyter/pull/45),
+  [swift-jupyter#48](https://github.com/google/swift-jupyter/pull/48),
+  [swift-jupyter#52](https://github.com/google/swift-jupyter/pull/52))
+* `swift-jupyter` can now be installed in a Conda environment. See
+  [documentation in
+  README](https://github.com/google/swift-jupyter#option-2-using-a-swift-for-tensorflow-toolchain-and-conda).
+
+### Swift Standard Library Enhancements
+
+* `AnyDerivative` has been added, representing a type-erased derivative.
+  ([apple/swift#23521](https://github.com/apple/swift/pull/23521))
+
+### TensorFlow Library
+
+* `Tensor` now supports advanced indexing and striding APIs.
+  ([apple/swift#24684](https://github.com/apple/swift/pull/23684))
+* `Tensor`s are now pretty-printed, based on the format of NumPy.
+  ([apple/swift#23837](https://github.com/apple/swift/pull/23837))
+* TensorFlow APIs now use `Int` instead of `Int32`.
+  ([apple/swift#24012](https://github.com/apple/swift/pull/24012))
+* The `saveV2` and `restoreV2` ops are now supported.
```

Comment:
I'd suggest explicitly stating they are raw operators (we are officially using the term "operator" instead of "op" also). Would you like to mention their TF name first, and their full Swift names? Something like:

"`SaveV2` (`Raw.saveV2(prefix:tensorNames:shapeAndSlices:tensors:)`), `RestoreV2` (`Raw.restoreV2(prefix:tensorNames:shapeAndSlices:dtypes:)`), `Split` (`Raw.split(splitDim:value:numSplit:)`) and `SplitV` (`Raw.splitV(value:sizeSplits:splitDim:numSplit:)`)".